app/controllers/display_tasks_controller.rb

```diff
@@ -1,12 +1,8 @@
 class DisplayTasksController < ApplicationController
```

Comment:
Recommend renaming this to be just TasksController based on our ruby naming conventions